documentation/API-design-guidelines.md

```diff
@@ -1183,16 +1198,16 @@ In order to ease developer adoption, the pattern for Resource-based event subscr
 
 | operation | path | description |
 | ----- |	-----  |	 -----  | 
-| POST | `/event-subscriptions` |  Operation to request a event subscription.     |
-| GET | `/event-subscriptions` |  Operation to retrieve a list of event subscriptions - could be an empty list.  eg. `GET /event-subscriptions?type=ROAMING_STATUS&ExpireTime.lt=2023-03-17` |
-| GET | `/event-subscriptions/{eventSubscriptionsId}` | Operation to retrieve a event subscription |
-| DELETE | `/event-subscriptions/{eventSubscriptionsId}` | Operation to delete a event subscription |
+| POST | `/subscriptions` |  Operation to request a event subscription.     |
+| GET | `/subscriptions` |  Operation to retrieve a list of event subscriptions - could be an empty list.  eg. `GET /subscriptions?type=ROAMING_STATUS&ExpireTime.lt=2023-03-17` |
```

Comment:
The type field value here probably needs to align with the new convention?

Reply:
Agreed - will fix.

Reply:
On one side alignment of subscription `type` and notification `type` is good for uniformity, but on the other side only notifications are covered by CloudEvents specification and requirements and using the reverse DNS notation for event subscription looks strange for me.
Using old simple notation in subscription makes another feature more intuitive: within one subscription events of different types can be generated, for example: Subscription to `type=ROAMING_STATUS` can result in `org.camaraproject.device-status.v1.roaming-on` or `org.camaraproject.device-status.v1.roaming-off` events.
For clarity we can distinguish parameter names and have subscription `kind`  and notification `type` as required by CloudEvents.
What are opinions of others?

Reply:
Hi @rartych, 
I agree with you, using "kind" in subscription might be more user friendly than using event type.
Of course we need more opinions.

Reply:
According to CloudEvents spec, the "type" value needs to be coherent among events and related subscriptions. Then, it is needed to have the same value and follow DNS-reverse format with the CAMARA pattern.

Reference:
https://github.com/cloudevents/spec/blob/main/subscriptions/spec.md#321-subscription-object

> Type: Array of Strings - array of CloudEvents type values

Description: Indicates which types of events the subscriber is interested in receiving. _When present on a subscribe request, all events generated due to this subscription MUST have a CloudEvents type property that matches one of these values._

Constraints:

OPTIONAL
If present, any value present MUST a non-empty string
Examples:

com.github.pull_request.opened
com.example.object.deleted